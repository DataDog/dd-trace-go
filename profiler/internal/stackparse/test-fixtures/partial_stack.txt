goroutine 1 [running]:
main.main()

goroutine 2 [running]:
main.main()
