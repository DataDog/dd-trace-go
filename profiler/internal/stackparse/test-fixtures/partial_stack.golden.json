{
  "Errors": [
    "invalid file:line ref on line 3: \"\"",
    "invalid file:line ref on line 6: \"\""
  ],
  "Goroutines": []
}
